lib/Github/Api/Integrations.php

```diff
@@ -2,107 +2,11 @@
 
 namespace Github\Api;
 
-use Github\Api\AcceptHeaderTrait;
-
 /**
- * @link   https://developer.github.com/v3/integrations/
+ * @deprecated
```

Comment:
Maybe we can do it "symfony" style with a `@trigger_error` when using the class. Like this https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/ClassLoader/ApcClassLoader.php#L14

But also keep the annotation so ide's will show it. What do you think @Nyholm?